frontend/src/pages/TermPlanner/UnplannedColumn/index.less

```diff
@@ -1,18 +1,75 @@
-.unplannedBox {
-  grid-row-start: 2;
+.unplannedContainer {
+  grid-row-start: 1;
   grid-row-end: span 10;
   grid-column-start: 5;
+  display: flex;
+  flex-direction: column;
+  place-self: stretch;
+  min-width: 15em;
+  align-items: center;
+}
+
+.summerUnplannedContainer {
+  grid-column-start: 6;
+}
+
+.unplannedTitle {
+  position: absolute;
+  font-weight: bold;
+  top: calc(
+    var(--navbar-height) + 
+    var(--option-header-height) + 
+    var(--tp-main-container-padding) + 
+    var(--tp-planner-container-margin) 
+  );
+}
+
+.unplannedBox {
+  position: absolute;
+  overflow-y: auto;
+  top: calc(
+    var(--navbar-height) + 
+    var(--option-header-height) + 
+    var(--tp-main-container-padding) + 
+    var(--tp-planner-container-margin) + 
+    var(--tp-term-box-margin) +
+    var(--tp-grid-item-font-size)
+  );
+  height: calc(
+    100vh - (
```

Comment:
oof alot of calcs haha. Not visually readable but I think this is the simpliest approach to do the unplanned column. Maybe could add a brief comment explaining why it's using calc?